components/hab/static/template_plan.sh

```diff
@@ -0,0 +1,12 @@
+pkg_name={{ pkg_name }}
+pkg_origin={{ pkg_origin }}
+pkg_version=0.0.1
+pkg_description="The package description"
```

Comment:
I think it'd be best for us to comment out everything except for the `pkg_name` & `pkg_origin`. Those are the only sensible defaults here 😉